src/fides/api/ctl/migrations/versions/6b9885e68cbb_add_plus_system_scans_table.py

```diff
@@ -0,0 +1,33 @@
+"""Add plus_system_scans table
```

Comment:
> is this the first time we're adding a db table that will only be used in plus? IMO it makes sense to keep the models and db (alembic migrations) in the same repo, but not sure if there's already been a separate pattern established.

This isn't the first time we've added tables/models to the core DB for use in -plus. See: https://github.com/ethyca/fides/pull/1060. We're avoiding adding any DB config to fidesctl-plus for as long as possible, because it will diverge the database management code and create a maintenance nightmare. It's less than ideal, but the "established pattern" is to do all DB maintenance in this repo, and just expose those otherwise unused tables to core users. The -plus codebase will leverage the core DB, and expect the required tables to exist.

> I believe after this PR goes in, anyone that does use alembic's autogenerate will generate a migration that will drop these tables, since they're not part of `sql_models.py`

I didn't realize that Alembic would automatically try to drop tables if the model wasn't listed in a specific file, nor did I realize the plus-only models were added to this repo in that PR I just linked. I went ahead and added a `SystemScans` model.